api/controllers/deployments.js

```diff
@@ -46,6 +46,7 @@ function postDeployment(req, res) {
         res.status(500).json({ "error": err.message});
       }
       statsdClient.increment("deployments.started");
+      statsdClient.increment("deployments.engine." + deployment.engine);
```

Comment:
Should we escape this with `statsdClient.escape()` as n the line below, in case there are dots or spaces in the engine name?